packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -830,16 +832,14 @@ class Viewport implements IViewport {
    * Updates the actors clipping planes orientation from the camera properties
    * @param updatedCamera - ICamera
    */
-  public updateActorsClippingPlanesOnCameraModified(
-    updatedCamera: ICamera
-  ): void {
+  public updateActorsClippingPlanes(updatedCamera: ICamera): void {
```

Comment:
- Any reason for this to be public? It seems like a private function.
- The method name is very weird to me due to the possessive plural in the middle. Maybe `updateClippingPlanesForActors`?